### PR TITLE
Truthy-Falsy values in  JS

### PR DESCRIPTION
**Pull request by ayushigeorge**
Implementation of Truthy and Falsy Values and Truthy and Falsy expresession

### CODE REVIEW

File: variable-in-js.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ const a=10;
+ function sqrt(){
+     const a=9;
+     console.log(a^a);
+ }
+ f();
+
+
+
+
```
Comment (sumitrathawani) [inline variable-in-js.js:40]: We have to add only one empty line at end of each file, and one line is required but not more than one, use prettier to format code properly before committing.

File: function-in-js.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+// Implemented Normal Function
+function sub(){
+    return a-b;
+}
+//Implemented Arrow Function 
+const div = (a,b) => a/b;
+
+const mod = (a,b)=> a%b;
```
Comment (sumitrathawani) [inline function-in-js.js]: add one empty line at end of each file. use prettier to format code properly

**Review by sumitrathawani — CHANGES_REQUESTED** [review]

File: variable-in-js.js (hunk 1 above)
Comment (ayushigeorge) [reply variable-in-js.js:40]: Resolved but not reflected `62db98a4ec37b`

File: function-in-js.js (hunk 2 above)
Comment (ayushigeorge) [reply function-in-js.js]: resolved issue `6b74bc8a80`

**Pull request closed without merging**